system/lib/standalone/standalone.c

```diff
@@ -155,7 +155,7 @@ size_t emscripten_get_heap_max() {
 }
 
 int emscripten_resize_heap(size_t size) {
-#ifdef EMSCRIPTEN_MEMORY_GROWTH
+#if defined EMSCRIPTEN_MEMORY_GROWTH && !defined EMSCRIPTEN_PURE_WASI
```

Comment:
I wasn't aware this notation worked. I am used to seeing
```suggestion
#if defined(EMSCRIPTEN_MEMORY_GROWTH) && !defined(EMSCRIPTEN_PURE_WASI)
```
Does it work either way?

Reply:
Yeah, either one work.  I switched to the more common form though.